lib/block_view_types.go

```diff
@@ -1821,7 +1878,42 @@ func (op *UtxoOperation) RawDecodeWithoutMetadata(blockHeight uint64, rr *bytes.
 		if exist, err := DecodeFromBytes(prevValidatorEntry, rr); exist && err == nil {
 			op.PrevValidatorEntry = prevValidatorEntry
 		} else if err != nil {
-			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevValidatorEntry")
+			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevValidatorEntry: ")
+		}
+
+		// PrevGlobalStakeAmountNanos
+		if prevGlobalStakeAmountNanos, err := ReadOptionalUint256(rr); err == nil {
+			op.PrevGlobalStakeAmountNanos = prevGlobalStakeAmountNanos
+		} else {
+			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevGlobalStakeAmountNanos: ")
+		}
+
+		// PrevStakeEntries
+		numPrevStakeEntries, err := ReadUvarint(rr)
+		if err != nil {
+			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevStakeEntries: ")
+		}
+		for ii := 0; ii < int(numPrevStakeEntries); ii++ {
+			prevStakeEntry := &StakeEntry{}
+			if exist, err := DecodeFromBytes(prevStakeEntry, rr); exist && err == nil {
+				op.PrevStakeEntries = append(op.PrevStakeEntries, prevStakeEntry)
+			} else if err != nil {
+				return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevStakeEntries: ")
+			}
+		}
```

Comment:
any maybe something like this for decoding? can slot this into a separate PR so it doesn't muddy this one.
```
func DecodeSlice[T DeSoEncoder](rr *bytes.Reader) ([]T, error) {
	numItems, err := ReadUvarint(rr)
	if err != nil {
		return nil, errors.Wrapf(err, "DecodeSlice: Problem decoding numItems")
	}
	// Note: is it more efficient to do a make with specific length and set at each index?
	inputs := make([]T, numItems)
	var results []T
	for ii := uint64(0); ii < numItems; ii++ {
		prevEntry := inputs[ii].GetEncoderType().New()
		if exist, err := DecodeFromBytes(prevEntry, rr); exist && err == nil {
			results = append(results, prevEntry.(T))
		} else if err != nil {
			return nil, errors.Wrapf(err, "DecodeSlice: Problem decoding item %d of %d", ii, numItems)
		}
	}
	return results, nil
}
```